<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>彩色化</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="sAeraCon_Image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>533</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="ScribbleArea" name="label_ShowImage">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_Operator" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCommandLinkButton" name="cLinBut_OpenFile">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>打开灰度文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="cLinBut_SelectScribble">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>打开上色文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="cLinBut_SaveFile">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>保存文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="cLinBut_DoColorization">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>图像彩色化</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="cLinBut_VideoColorization">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>视频彩色化</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_Sep">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="cLinBut_SelectCorlor">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>选择颜色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_Color" native="true">
           <property name="toolTip">
            <string>画笔颜色</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_ShowColor">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_LineWidth" native="true">
           <property name="toolTip">
            <string>画笔线宽</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item alignment="Qt::AlignLeft">
             <widget class="QSpinBox" name="spinBox_LineWidth">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_ShowLineWidth">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>1</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>10</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>0</width>
                   <height>5</height>
                  </size>
                 </property>
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="cLinBut_ClearScribble">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>清除着色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="cLinkBut_MethodSelection">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>算法选择</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>117</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScribbleArea</class>
   <extends>QLabel</extends>
   <header>scribblearea.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
